Sources/ArgumentParser/Usage/HelpGenerator.swift

```diff
@@ -76,8 +88,14 @@ internal struct HelpGenerator {
       guard !elements.isEmpty else { return "" }
       
       let renderedElements = elements.map { $0.rendered(screenWidth: screenWidth) }.joined()
-      return "\(String(describing: header).uppercased()):\n"
-        + renderedElements
+
+      var renderedHeader = String(describing: header).uppercased() + ":"
+      if let abstract = header.abstract {
+        renderedHeader += " "
+        renderedHeader += abstract.wrapped(to: screenWidth)
```

Comment:
Do we need to add the abstract and then wrap?

Reply:
Ahhh, probably. Thank you